packages/react-components/react-motion/library/src/hooks/useAnimateAtoms.ts

```diff
@@ -19,10 +22,10 @@ function useAnimateAtomsInSupportedEnvironment() {
           fill: 'forwards',
 
           ...params,
-          ...(isReducedMotion && { duration: 1 }),
+          ...((isReducedMotion || !SUPPORTS_PERSIST) && { duration: 1 }),
```

Comment:
Why do you need to go through the reduced motion path? the animation will still work without `persist` - the main consequence is that with for multiple animations that target the same property, the latest animation will replace older ones

Reply:
I don't strictly need to. My thinking is twofold:

1. It reduces the likelihood of multiple animations targeting the same property. I don't know if this is really true in practice but if you had multiple, delayed animations it seems like it would be.
2. If your browser doesn't support `.persist()` you don't really fully support this feature so we just opt you out of it. Using reduced motion ensures the functionality is observably the same (barring the multiple animation situation) as all animation events will fire, etc. This only affects browsers in our partial support matrix so a graceful degradation approach feels fine to me.

Open to changing this but as-is seems best to me.